src/sys/wait.rs

```diff
@@ -42,11 +55,15 @@ const WSTOPPED: WaitPidFlag = WUNTRACED;
 
 #[derive(Eq, PartialEq, Clone, Copy, Debug)]
 pub enum WaitStatus {
+    /// Signifies that the process has exited, providing the PID and associated exit status.
     Exited(Pid, i8),
+    /// Signifies that the process was killed by a signal, providing the PID and associated signal.
     Signaled(Pid, Signal, bool),
+    /// Signifies that the process was stopped by a signal, providing the PID and associated signal.
     Stopped(Pid, Signal),
     #[cfg(any(target_os = "linux", target_os = "android"))]
     PtraceEvent(Pid, Signal, c_int),
+    /// Signifies that the process received a `SIGCONT` signal, and thus continued.
     Continued(Pid),
     StillAlive
```

Comment:
I'd like to get some doc comments on `StillAlive` and `PtraceEvent` since we're here. Are you stuck on understanding those or did you just forget to add them?